src/inject/GPMInject/interface/toast.js

```diff
@@ -9,6 +9,7 @@ Emitter.on('toast:pauseAfter', () => {
   disableButton.innerHTML = 'Don\'t pause after this song';
   disableButton.style.color = '#E53935';
   disableButton.style.margin = '0';
+  disableButton.style.marginBottom = '102px';
```

Comment:
`bottom` not `marginBottom`, as it's absolutely positioned in the lower left corner.